wurst/systems/modes/Quests.wurst

```diff
@@ -15,6 +15,7 @@ let coolCommands = asList(
     "-rn:|r shows the tribe per player".color(HIGHLIGHT_COLOR),
     "-sd:|r Enable/disable damage display".color(HIGHLIGHT_COLOR),
     "-ms:|r displays your troll's movement speed".color(HIGHLIGHT_COLOR),
+    "-dps #attack speed #damage:|r displays your troll's attack speed & DPS".color(HIGHLIGHT_COLOR),
```

Comment:
The description for `-as` was never displayed here, so it is not in the diff even though the command is deleted.